src/BalanceTx.purs

```diff
@@ -532,13 +532,11 @@ logTx msg utxos (Transaction { body: body'@(TxBody body) }) =
     , "Fees: " <> show body.fee
     ]
 
--- Nami provides a 5 Ada collateral that we should add the tx before balancing
+-- Nami provides a 5 Ada collateral that we should add the tx before balancing.
```

Comment:
```
- -- Nami provides a 5 Ada collateral that we should add the tx before balancing.
+ -- Nami provides a 5 Ada collateral that we should add to the tx before balancing.
```

?

Reply:
@adamczykm Hm, just added a period to the end of the docstring :)